helper/hex/hex_test.go

```diff
@@ -0,0 +1,42 @@
+package hex
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestEncodingUint64(t *testing.T) {
+	testTable := []struct {
+		name       string
+		value      uint64
+		shouldFail bool
+	}{
+		{
+			"Successfully encode and decode uint64",
+			245,
+			false,
+		},
+		{
+			"Successfully encode and decode uint64",
+			0,
+			false,
+		},
+	}
+
+	for _, testCase := range testTable {
+		t.Run(testCase.name, func(t *testing.T) {
+			encodedValue := EncodeUint64(testCase.value)
+
+			decodedValue, err := DecodeUint64(encodedValue)
+			if err != nil && !testCase.shouldFail {
+				t.Fatalf("Unable to decode uint64, %v", err)
+			}
+
+			assert.Equal(t, testCase.value, decodedValue)
+
+			decodedValue = MustDecodeUint64(encodedValue)
+
+			assert.Equal(t, testCase.value, decodedValue)
```

Comment:
This is redundant, since `MustDecodeUint64` is redundant

Reply:
fixed a50aef5ac33eb44c76d87c8367a20a1158c146fe